worker/wallet.js

```diff
@@ -227,6 +229,12 @@ async function checkWithdrawal ({ data: { hash }, boss, models, lnd }) {
     }
   }
 
+  let walletName
+  if (dbWdrwl.wallet) {
+    const { wallet } = dbWdrwl.wallet
+    walletName = wallet.address ? 'walletLNAddr' : wallet.macaroon ? 'walletLND' : 'walletCLN'
```

Comment:
This works, but dbWdrwl.wallet.type is probably be a better way to determine wallet types.

Reply:
Oh, yes

Reply:
Done in https://github.com/stackernews/stacker.news/pull/1073/commits/3d4a15dbdc66644a5c259eb69f88dbe81374c2bd

Reply:
🙊 I had a code review draft that mentioned that change in the wallet log PR but figured it was better to let you find it. I figured so long as it's not going to break, I can take this approach. :)

Reply:
This is actually a fun game to play :eyes: 

> _What did @huumn not tell me about this PR?_

Reply:
This is why shipping MVPs is good because you begin thinking about what people might notice. Same goes for open source code. You begin simulating accountability.

Reply:
This reminds me of this:

> I believe our brains create a literal simulation of our partner's mind that overtime converges towards their actual thought processes. In that way, we are feeling and thinking what they're feeling and thinking, and vice versa.

-- https://stacker.news/items/413755

Your analogy with startup hires being like marriages becomes more and more true over time